website/content/docs/ecs/get-started/install.mdx

```diff
@@ -0,0 +1,407 @@
+---
+layout: docs
+page_title: Install - AWS ECS
+description: >-
+  Install Consul Service Mesh on AWS ECS (Elastic Container Service).
+---
+
+# Install
+
+Installing Consul on ECS is a multi-part process:
+
+1. [**Terraform:**](#terraform) Your tasks must be specified in Terraform using [`ecs_task_definition`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecs_task_definition)
+   and [`ecs_service`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecs_service) resources.
+1. [**Consul Server:**](#consul-server) You must deploy the Consul server onto the cluster using the [`dev-server` module](https://registry.terraform.io/modules/hashicorp/consul/aws-ecs/latest/submodules/dev-server).
+1. [**Task IAM Role:**](#task-iam-role) Modify task IAM role to add `ecs:ListTasks` and `ecs:DescribeTasks` permissions.
+1. [**Task Module:**](#task-module) You can then take your `ecs_task_definition` resources and copy their configuration into a new [`mesh-task` module](https://registry.terraform.io/modules/hashicorp/consul/aws-ecs/latest/submodules/mesh-task)
+   resource that will add the necessary containers to the task definition.
+1. [**Routing:**](#routing) With your tasks as part of the mesh, you must specify their upstream
+   services and change the URLs the tasks are using so that they're making requests
+   through the service mesh.
+1. [**Bind Address:**](#bind-address) Now that all communication is flowing through the service mesh,
+   you should change the address your application is listening on to `127.0.0.1`
+   so that it only receives requests through the sidecar proxy.
+
```

Comment:
It might help those looking on this page to add a note that this guide assumes you know ECS and point to [documentation](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/Welcome.html) if not.

Reply:
Done!